package.json

```diff
@@ -15,7 +15,8 @@
   },
   "phenomic": {
     "CNAME": false,
-		"offline": true
+    "offline": true,
+    "clientScripts": false
```

Comment:
sorry, that's the config for service workers, line 19 is what I think will fix this particular problem